crt_portal/cts_forms/filters.py

```diff
@@ -90,11 +90,13 @@ def report_grouping(querydict):
     all_qs, filters = report_filter(querydict)
     for group in groups:
         description = group['violation_summary']
-        summaries.append(description)
-        group_queries.append({
-            "qs": all_qs.filter(violation_summary=description),
-            "desc": description
-        })
+        qs = all_qs.filter(violation_summary=description)
+        if qs.count() > 1:
+            summaries.append(description)
+            group_queries.append({
+                "qs": all_qs.filter(violation_summary=description),
+                "desc": description
+            })
```

Comment:
aim to [reduce nesting using guard clauses](testing.googleblog.com/2017/06/code-health-reduce-nesting-reduce.html)

```suggestion
        qs = all_qs.filter(violation_summary=description)
        if qs.count() <= 1:
            continue
        summaries.append(description)
        group_queries.append({
            "qs": all_qs.filter(violation_summary=description),
            "desc": description,
        })
```